### PR TITLE
Add a checkbox to fixtures UI to choose React production build

### PR DESCRIPTION
**Pull request by poeschko**
This started out as a part of #13613 which will probably be closed.

### CODE REVIEW

**Comment by poeschko** [comment]
@philipp-spiess kindly [deployed](https://github.com/facebook/react/pull/13613#issuecomment-423336433) this for taking a quick look: https://philipp-react-dom-fixtures.netlify.com/

File: fixtures/dom/src/components/Header.js [hunk 1]
```diff
@@ -43,6 +52,15 @@ class Header extends React.Component {
...
 
           <div className="header-controls">
+            <input
+              id="react_production"
+              type="checkbox"
+              checked={this.state.production}
+              onChange={this.handleProductionChange}
+            />
+            <label htmlFor="react_production">
+              <span className="header__label">Production</span>
```
Comment (nhunzaker) [inline fixtures/dom/src/components/Header.js]: Could you eliminate this extra span by adding the class name to the label?

File: fixtures/dom/src/components/Header.js [hunk 2]
```diff
@@ -43,6 +52,15 @@ class Header extends React.Component {
           </span>
 
           <div className="header-controls">
+            <input
+              id="react_production"
+              type="checkbox"
+              checked={this.state.production}
+              onChange={this.handleProductionChange}
+            />
```
Comment (nhunzaker) [inline fixtures/dom/src/components/Header.js]: Could you add a style to center this vertically with the label? maybe:

```css
.header__checkbox {
  vertical-align: middle;
}
```

**Comment by nhunzaker** [comment]
@poeschko Hey! Just wanted to check in. I'd be happy to move this along if you'd like.

File: fixtures/dom/src/components/Header.js (hunk 1 above)
Comment (poeschko) [reply fixtures/dom/src/components/Header.js]: Yes, good idea.

File: fixtures/dom/src/components/Header.js (hunk 2 above)
Comment (poeschko) [reply fixtures/dom/src/components/Header.js]: Yes, done.

**Comment by poeschko** [comment]
Thanks for the comments (which I have addressed now), and sorry for the delay.

**Comment by nhunzaker** [comment]
Awesome, thank you! I'm going to do a quick build locally, but this is ready to 🚢!

**Comment by nhunzaker** [comment]
Checks out. Awesome stuff, thanks for doing this!

**Comment by philipp-spiess** [comment]
Thanks for reviewing, @nhunzaker 🎉

**Pull request merged**